src/Rx/Weno/wenoconnected.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
this script should be moved to interface subdirectory.
Would put it in:
interface/weno/

Reply:
done